code/modules/mob/mob_movement.dm

```diff
@@ -15,21 +15,27 @@
 			return 1
 	return (!mover.density || !density || lying)
 
+//The byond version of these verbs wait for the next tick before acting.
+//	instant verbs however can run mid tick or even during the time between ticks.
+/client/verb/moveup()
+	set name = ".moveup"
+	set instant = 1
+	Move(get_step(mob, NORTH), NORTH)

```

Comment:
I'll keep this in mind, but at the moment we flat don't support client.dir - many different sprites and visual effects completely break from using it.